### PR TITLE
feat(documentation): add content for the welcome page

### PR DESCRIPTION
**Pull request by oliverschuerch**

### CODE REVIEW

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

File: packages/documentation/src/stories/welcome.stories.mdx [hunk 1]
```diff
@@ -12,14 +15,203 @@ import { Meta } from '@storybook/addon-docs';
...
+  </a>
+</div>
+
+  <p className="fs-tiny text-muted">
+    <sup>1</sup> login required
+  </p>
+</div>
+
+<div className="mt-huge py-huge bg-light bg-fullwidth text-center">
+  <h2 className="h3 mb-big">Reasons to choose the Swiss Post Design System</h2>
```
Comment (gfellerph) [inline packages/documentation/src/stories/welcome.stories.mdx:108]:
```suggestion
  <h2 className="h3 mb-big">Features</h2>
```

File: packages/documentation/src/stories/welcome.stories.mdx [hunk 2]
```diff
@@ -12,14 +15,203 @@ import { Meta } from '@storybook/addon-docs';
...
+    <div className="col-md-4">
+      <Benefit icon="3161" title="Free">
+        <p>Open sourced on <a href="https://github.com/swisspost/design-system" target="_blank" rel="noopener">GitHub</a>, <a href="https://github.com/swisspost/design-system/blob/main/LICENSE" target="_blank" rel="noopener">Apache License</a></p>
+      </Benefit>
+    </div>
+  </div>
+</div>
+
+<div className="mt-huge text-center">
+  <h2 className="h3 mb-big">Technology Stack</h2>
```
Comment (gfellerph) [inline packages/documentation/src/stories/welcome.stories.mdx:145]:
```suggestion
  <h2 className="h3 mb-big">Technologies</h2>
```

File: packages/documentation/src/stories/welcome.stories.mdx [hunk 3]
```diff
@@ -12,14 +15,203 @@ import { Meta } from '@storybook/addon-docs';
...
+    <span>v{getVersion('icons')}</span>
+  </Tile>
+</div>
+
+<div className="mt-huge mb-regular d-flex gap-mini flex-wrap justify-content-center">
+  {/* <div className="w-100 mb-regular">
+    <a href="#" className="btn btn-primary btn-lg">
+      Get started
+    </a>
+  </div> */}
```
Comment (gfellerph) [inline packages/documentation/src/stories/welcome.stories.mdx:63]: Add a todo page "Getting started" and include the primary button again or remove the commented out code.

File: packages/documentation/src/stories/welcome.stories.mdx [hunk 4]
```diff
@@ -12,14 +15,203 @@ import { Meta } from '@storybook/addon-docs';
...
-    background: 'red',
-    fontWeight: 'bold',
-    color: 'white',
-  }}
->
-  TODO: write welcome page
+<div className="mt-huge text-center">
+  <h2 className="h3 mb-big">Packages</h2>
+
+<p>We offer different packages, which can all be used together or individually.</p>
```
Comment (gfellerph) [inline packages/documentation/src/stories/welcome.stories.mdx:21]:
```suggestion
```

**Review by gfellerph — CHANGES_REQUESTED** [review]

File: packages/documentation/src/stories/home.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+        <p><a href="https://www.w3.org/WAI/standards-guidelines/aria/" target="_blank" rel="noopener">WAI-ARIA accessibility</a> markup</p>
+      </Feature>
+    </div>
+    <div className="col-md-4">
+      <Feature icon="2483" title="Modern">
+        <p>Built with <a href="https://stenciljs.com/" target="_blank" rel="noopener">Stencil JS</a> and <a href={ `https://getbootstrap.com/docs/${getVersion('bootstrap', 'Mm')}` } target="_blank" rel="noopener">Bootstrap</a></p>
+      </Feature>
+    </div>
+    <div className="col-md-4">
+      <Feature icon="3161" title="Free">
```
Comment (gfellerph) [inline packages/documentation/src/stories/home.stories.mdx:140]:
```suggestion
      <Feature icon="2444" title="Open Source">
```

File: packages/documentation/src/stories/home.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+        <p>Open sourced on <a href="https://github.com/swisspost/design-system" target="_blank" rel="noopener">GitHub</a>, <a href="https://github.com/swisspost/design-system/blob/main/LICENSE" target="_blank" rel="noopener">Apache License</a></p>
```
Comment (gfellerph) [inline packages/documentation/src/stories/home.stories.mdx:141]:
```suggestion
        <p>Published on <a href="https://github.com/swisspost/design-system" target="_blank" rel="noopener">GitHub</a> under the <a href="https://github.com/swisspost/design-system/blob/main/LICENSE" target="_blank" rel="noopener">Apache 2.0 license</a></p>
```

File: packages/documentation/src/stories/home.stories.mdx [hunk 7]
```diff
@@ -0,0 +1,220 @@
...
+      </Feature>
+    </div>
+    <div className="col-md-4">
+      <Feature icon="2115" title="Accessible">
+        <p><a href="https://www.w3.org/WAI/standards-guidelines/aria/" target="_blank" rel="noopener">WAI-ARIA accessibility</a> markup</p>
+      </Feature>
+    </div>
+    <div className="col-md-4">
+      <Feature icon="2483" title="Modern">
+        <p>Built with <a href="https://stenciljs.com/" target="_blank" rel="noopener">Stencil JS</a> and <a href={ `https://getbootstrap.com/docs/${getVersion('bootstrap', 'Mm')}` } target="_blank" rel="noopener">Bootstrap</a></p>
```
Comment (gfellerph) [inline packages/documentation/src/stories/home.stories.mdx:136]:
```suggestion
        <p>Using modern web standards like framework agnostic web components</p>
```

File: packages/documentation/src/stories/home.stories.mdx [hunk 8]
```diff
@@ -0,0 +1,220 @@
...
+      </Feature>
+    </div>
+    <div className="col-md-4">
+      <Feature icon="2097" title="Modular">
+        <p>Import only the features that you need</p>
+      </Feature>
+    </div>
+    <div className="col-md-4">
+      <Feature icon="2115" title="Accessible">
+        <p><a href="https://www.w3.org/WAI/standards-guidelines/aria/" target="_blank" rel="noopener">WAI-ARIA accessibility</a> markup</p>
```
Comment (gfellerph) [inline packages/documentation/src/stories/home.stories.mdx:131]:
```suggestion
        <p>Following the <a href="https://www.w3.org/TR/WCAG21/" target="_blank" rel="noopener">WCAG 2.1</a> guidelines</p>
```

File: packages/documentation/src/stories/home.stories.mdx [hunk 9]
```diff
@@ -0,0 +1,220 @@
...
+  <p className="mb-0 fs-tiny lh-tiny text-muted">
+    <sup>2</sup> Login required. Get access with your post account, or click <a href="mailto:ux@post.ch?subject=I need access to Figma&body=Hello%2C%20I%20need%20access%20to%20Figma%20%0A%E2%80%A2%20%20Project%20name%3A%20%0A%E2%80%A2%20Project%20Owner%20at%20the%20Post%20Office%20%28email%29%3A%20%0A%E2%80%A2%20Project%20Start%20Date%3A%20%0A%E2%80%A2%20Email%20address%20to%20be%20authorized%3A%20">here</a>.
+  </p>
+</div>
+
+<div className="mt-huge text-center">
+  <LinkTo kind="Get Started" className="btn btn-primary btn-lg">
+    Get started
+  </LinkTo>
+</div>
```
Comment (gfellerph) [inline packages/documentation/src/stories/home.stories.mdx:108]: I'd put that above the secondary buttons/links as it's more important.

**Review by gfellerph — CHANGES_REQUESTED** [review]

File: packages/documentation/src/stories/home.stories.mdx [hunk 10]
```diff
@@ -0,0 +1,224 @@
...
+      canvas: {
+        hidden: true,
+      },
+    },
+    viewMode: 'docs',
+  }}
+/>
+
+<div className="mt-huge text-center">
+  <h2 className="h3">Packages &amp; Rescources</h2>
```
Comment (gfellerph) [inline packages/documentation/src/stories/home.stories.mdx:20]:
```suggestion
  <h2 className="h3">Packages &amp; Resources</h2>
```

**Review by gfellerph — CHANGES_REQUESTED** [review]
Just a small typo left

**Review by gfellerph — APPROVED** [review]

**Pull request merged**